### PR TITLE
July k3s release

### PR DESCRIPTION
**Pull request by nikolaishields**

### CODE REVIEW

**Review by rancher-max — CHANGES_REQUESTED** [review]
Should be `+k3s1` instead of k3s2 for all of these

File: channels.yaml [hunk 1]
```diff
@@ -219,6 +219,12 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
+  - version: v1.22.12-rc1+k3s2
```
Comment (VestigeJ) [inline channels.yaml]: same fix with s1 vs s2 @nikolaishields

File: channels.yaml [hunk 2]
```diff
@@ -242,9 +248,21 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
+  - version: v1.23.9-rc1+k3s2
```
Comment (VestigeJ) [inline channels.yaml]: same fix with s1 vs s2

File: data/data.json [hunk 3]
```diff
@@ -14900,6 +14900,161 @@
...
+      "type": "string"
+     },
+     "service-node-port-range": {
+      "type": "string"
+     },
+     "tls-san": {
+      "type": "array"
+     }
+    },
+    "version": "v1.22.12-rc1+k3s2"
```
Comment (VestigeJ) [inline data/data.json:15056]: v1.22.12-rc1+k3s1 ** @nikolaishields

File: data/data.json [hunk 4]
```diff
@@ -15658,7 +15813,317 @@
       "type": "array"
      }
     },
+    "version": "v1.23.9-rc1+k3s2"
```
Comment (VestigeJ) [inline data/data.json:15816]: v1.23.9-rc1+k3s1 **

**Review by VestigeJ — DISMISSED** [review]
LGTM thank you @nikolaishields

**Review by rancher-max — DISMISSED** [review]
LGTM

**Comment by kinarashah** [comment]
https://github.com/rancher/rancher/issues/38258

**Review by brandond — DISMISSED** [review]

**Comment by nikolaishields** [comment]
August PR facilitates these changes.

**Pull request closed without merging**